nxt/system.py

```diff
@@ -217,7 +217,7 @@ def get_device_info(opcode):
 
 def _parse_get_device_info(tgram):
     tgram.check_status()
-    name = tgram.parse_string(15)
+    name = tgram.parse_string(15).decode('utf-8').split('\0')[0]
```

Comment:
The thing that worries me about using `'utf-8'` is that if we get a non-ASCII character (in the range 0x80 to 0xff), then this will raise an exception.

```python
>>> b'\xfd'.decode()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xfd in position 0: invalid start byte
```

I use 0xfd as an example because the [wiki](https://github.com/Eelviny/nxt-python/wiki/Digital-Sensor-Guide#checking-for-sensor-compatibility) show a real example of when this character is returned.